tools/pkg/config/config.go

```diff
@@ -14,6 +14,7 @@ type Config struct {
 	Repositories []Repository
 }
 
+// Image should not be instantiated directly. Instead, use NewImage().
```

Comment:
Shouldn't it be private in this case? I.e. `image` instead of `Image`?

Reply:
I don't think so - we need to be able to pass `Image`s to functions, and return them from functions (for example, `convertImageListEntryToImage()`), which requires that the type is public.